### ISSUE TITLE
Product card component

### ISSUE THREAD

**Issue by r4pt0s** [zero-to-mastery/WebBlocks#4]
As the title already suggests, a product car component.
It's more like a case study how far we can get here with such a component.
Chances are that this one component contains more other components as well.
At least it should get split into smaller chunks which can totally happen in another PR then.

Feel free to come up with features and design yourself.

**Comment by pri1311** [comment]
I would like to work on this, however, I had a few questions as to how detailed it has to be? I assume it would be the title, cost, description, and image. Is there anything else you expect?

**Comment by r4pt0s** [comment]
@pri1311
No, for the first iteration of this component, this would be totally fine.

For the future I have things in mind like
- a different image on hover or a text which appears on hover
- add to cart action
- product variation swatches like we all know it from Amazon. If you click on one of them, the color of the product changes for example.

Shall I assign it to you?

**Comment by pri1311** [comment]
I believe implementing the first two features you mentioned wouldn't be difficult, however, I believe product variation is beyond my scope.

> Shall I assign it to you?

Yes, would like to work on this one.

**Comment by r4pt0s** [comment]
No worries @pri1311 you don't have to.

Going with the basics is absolutely fine.
Go for it 💪🏻

**Comment by StrykerKent** [comment]
Is this still being worked on?

If not, I would like to take a shot at it.

**Comment by r4pt0s** [comment]
@pri1311
Did you develop anything for this component till now?

**Comment by pri1311** [comment]
@r4pt0s I couldn't complete the component. I Will unassign myself.
